docs/source/traitsui_user_manual/testing/references/examples.rst

```diff
@@ -0,0 +1,21 @@
+
+Examples
+========
+
+Several test examples can be found for testing
+:ref:`TraitsUI\'s own demos<traitsui-demo>`.
+
+Editors
+-------
+- :github-demo:`BooleanEditor <Standard_Editors/tests/test_BooleanEditor_simple_demo.py>`
+- :github-demo:`ButtonEditor <Standard_Editors/tests/test_ButtonEditor_simple_demo.py>`
+- :github-demo:`CheckListEditor <Standard_Editors/tests/test_CheckListEditor_simple_demo.py>`
+- :github-demo:`EnumEditor <Standard_Editors/tests/test_EnumEditor_demo.py>`
+- :github-demo:`InstanceEditor <Standard_Editors/tests/test_InstanceEditor_demo.py>`
+- :github-demo:`ListEditor <Advanced/tests/test_List_editor_notebook_selection_demo.py>`
```

Comment:
Note there is also https://github.com/enthought/traitsui/blob/master/traitsui/examples/demo/Standard_Editors/tests/test_ListEditor_demo.py 
But the List example you have linked is probably the better example to refer to

Reply:
I did not notice that. Thank you. I think they are both good and should be included. The standard one probably should go first.